pkg/curatedpackages/common.go

```diff
@@ -0,0 +1,16 @@
+package curatedpackages
+
+import (
+	"context"
+
+	"github.com/aws/eks-anywhere/pkg/dependencies"
+	"github.com/aws/eks-anywhere/pkg/executables"
+)
+
+func createKubectl(ctx context.Context) (*dependencies.Dependencies, error) {
```

Comment:
Idiomatic Go says name this "newDependencies".

Reply:
This is also only used in `bundle.go` so move it there and drop this source file.